Code/Engine/Foundation/Strings/StringView.h

```diff
@@ -25,13 +25,13 @@ struct EZ_FOUNDATION_DLL ezStringView : public ezStringBase<ezStringView>
   ezStringView();
 
   /// \brief Creates a string view starting at the given position, ending at the next '\0' terminator.
-  ezStringView(const char* pStart); // [tested]
+  constexpr ezStringView(const char* pStart); // [tested]
 
   /// \brief Creates a string view for the range from pStart to pEnd.
   ezStringView(const char* pStart, const char* pEnd); // [tested]
```

Comment:
I guess making this one constexpr isn't necessary?

Reply:
It is neccessary. Otherwise `constexpr auto someConstant = "Hello World"_ezsv;` doesn't work.